thunder/distributed/__init__.py

```diff
@@ -278,6 +278,9 @@ class FSDPBucketingStrategy(Enum):
 def get_extract_bucket_name_from_tensor_proxy(granularity: FSDPBucketingStrategy):
     from thunder.core.proxies import TensorProxy
 
+    # TODO(crcrpar): Consider making `BLOCK`'s behavior more meticulous for models with simple structure
```

Comment:
unrelated to the PR but good example, @t-vi @lantiga shall we create issues for all these todos in code, so it won't be lost, I bet (at least me) would not come back to one todos in code :)